apps/condo/domains/billing/schema/BillingIntegrationOrganizationContext.js

```diff
@@ -74,6 +80,38 @@ const BillingIntegrationOrganizationContext = new GQLListSchema('BillingIntegrat
             },
         },
 
+        integrationOption: {
```

Comment:
Why not place this data to settings ?

Reply:
BillingIntegrationOrganiationContext.settings

We use this field specifically for this and it is not changed

Reply:
settings is more about keys, links.....generally speaking, it's using like .env

But I think it's not good idea to store billing format together :(